types/pure.d.ts

```diff
@@ -1,13 +1,14 @@
 import type {SpawnOptionsWithoutStdio} from 'child_process'
 import {ChildProcessWithoutNullStreams} from 'child_process'
-import type userEvent from '../src/user-event'
-import type {queries, BoundFunction} from './get-queries-for-instance'
+import type userEvent from '../src/user-event/index'
+import * as queries from './queries'
+import type {BoundFunction} from './get-queries-for-instance'
 
 export interface TestInstance {
   clear(): void
   process: ChildProcessWithoutNullStreams
-  stdoutArr: Array<string, Buffer>
-  stderrArr: Array<string, Buffer>
+  stdoutArr: Array<string | Buffer>
+  stderrArr: Array<string | Buffer>
```

Comment:
`Array<T>` takes a single generic argument, reading the `pure.ts` file I figured out you wanted a union type here.

Reply:
Doh! You're totally right